attacked/non_update_functions.py

```diff
@@ -0,0 +1,21 @@
+import functions
+import inspect
+from importlib import reload
+
+
+def update(request_params: str) -> str:
+    with open("functions.py", "w") as file:

```

Comment:
why updating. consider using path that is relative to the current running module and not relative to the CWD